frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -346,12 +353,27 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
                         <Field name="conditions">
                             {({ value, onChange }) => (
                                 <>
-                                    <PureField label="URL contains:">
-                                        <LemonInput
-                                            value={value?.url}
-                                            onChange={(urlVal) => onChange({ ...value, url: urlVal })}
-                                            placeholder="ex: https://app.posthog.com"
-                                        />
+                                    <PureField label="URL targeting">
+                                        <div className="flex flex-row gap-2 items-center">
+                                            URL
+                                            <LemonSelect
```

Comment:
This won't work for users unless they upgrade to latest posthog-js, so maybe we can include some sort of tooltip or message somewhere about it? 🤔